src/List.h

```diff
@@ -131,6 +131,9 @@ template <typename T, ListOrder Order = ListOrder::ORDERED> class List
 	// Return nth ent of list (do not remove).
 	T& operator[](int i) const { return entries[i]; }
 
+	T& at(int i) { return entries[i]; }
+	const T& at(int i) const { return entries[i]; }
```

Comment:
At least in the STL, the `at` operation typically does a bound-check. This does not seem to do this. Is there a reason to also have this in addition to the `[]` operator?

I might just be missing something here.

Reply:
It just makes it possible to do t->at(i) instead of having to do (*t)[i]. It results in the same code either way, so maybe I'm just being anal about how it looks.

Reply:
FWIW -as an unsuspecting user, I think I might expect the operator to do bounds-checking, mostly based on the fact that the stl does it for all at operators.

That being said - I don't have super strong feelings about it.

Reply:
Yep, I was just looking at `std::vector`. `at()` does bounds checking, where as `operator[]` doesn't. I'll just take it out 😃